docker-compose example/example.env

```diff
@@ -62,3 +62,10 @@ WATCHTOWER_POLL_INTERVAL=86400
 WATCHTOWER_TIMEOUT=30s
 WATCHTOWER_CLEANUP=true
 ##\\\\\\\\\\\\\\\\\//////////////////##
+
+##----------Video variables----------##
+# Set the hardware acceleration when encoding videos.
+# Support `qsv`, `vaapi`, `nvenc`.
+# Only `qsv` is verified with `/dev/dri//dev/dri` devices.
+# PHOTOVIEW_VIDEO_HARDWARE_ACCELERATION=
```

Comment:
I think that this variable has to be referenced in the compose file(s) in order to be accessible inside the container.

Please put this section right after the `Photoview variables` section, as it relates to the Photoview service, while other sections are for other services. Does it make sense to include the variable with its description inside the `Photoview variables` section?
BTW, text explanations are prefixed with `##`, while optional variables - with `#`. Please follow this style.

Apart from that, to use the HW acceleration feature, a user needs to configure the env by not only setting the acceleration type in the `.env`, but also making changes in the compose file and on the host.
Can you please add a brief description of what has to be done to the Readme, and corresponding comments, and changes to the compose file? Of course, you can describe the config path for QSV, but also please put links or short descriptions for other supported types.

Reply:
Do you have a better idea to organize documents? I think current README is already pretty long, and putting all features in one file is not easy to maintain.

One idea is creating a `doc` directroy, and link it in the README. But I don't have a good idea for [the website]([photoview.github.io/](https://photoview.github.io/)).

Reply:
Updated the session location and example Docker compose file.

I can't find a proper place in README to describe this feature. As README mentions [`docker-compose.example.yml`](https://github.com/photoview/photoview?tab=readme-ov-file#getting-started--setup-with-docker) to set it up, I added the description in it, just above the device session.

Reply:
> I can't find a proper place in README to describe this feature

Here is an example of how it could be included from my old branch: https://github.com/kkovaletp/photoview/blob/hw-gpu-support/README.md#advanced-setup

The main idea is to provide users with a short description of the feature, clear instructions on how to configure it end-to-end, a list of known limitations, and links to the external guides for not-tested configs. Try to wear the shoes of a user, who is completely out of the feature context and got notified about this possibility from the release notes. He/she likes the idea of enabling the HW acceleration but doesn't know what to do. Should anything be done on the host, Docker, or product config?

Reply:
Done.